bin/install-protobuf.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/sh
 set -e
 # check to see if protobuf folder is empty
-if [ ! -d "$HOME/protobuf/lib" ]; then
-  curl -L -o protobuf-3.3.0.tar.gz https://github.com/google/protobuf/archive/v3.3.0.tar.gz
-  tar -xzvf protobuf-3.3.0.tar.gz
-  cd protobuf-3.3.0 && ./autogen.sh && ./configure --prefix=$HOME/protobuf && make && make install
+download_url='https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip'
+unamestr=`uname`
+echo $unamestr
+if [ "$unamestr" == 'Darwin' ]; then
+   download_url='https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-osx-x86_64.zip'
+fi
+if [ ! -d "$HOME/protobuf" ]; then
+  curl -L -o protoc-3.3.0.zip $download_url
+  unzip -d protoc-3.3.0 -a protoc-3.3.0.zip
+  mv -v ./protoc-3.3.0 ~/protobuf
+  rm protoc-3.3.0.zip
```

Comment:
1 more change would make this cherry....    would it make sense to pull the version out as a variable so that it could be easily changed once.     var="3.3.0"

Reply:
Even better... make it an argument, and default it to 3.3.0?

Extra mile, put the version in a property file, source it in this script, and then update sbt to read the same property file?

Reply:
good point, I made it parameter with default for now...